docs/sources/get-started/quick-start.md

```diff
@@ -64,124 +49,137 @@ It's a fully configured environment with all the dependencies already installed.
-   ```
-   {{< /tab-content >}}
-   {{< /tabs >}}
-   <!-- INTERACTIVE ignore END -->
-
-   {{< docs/ignore >}}
-   ```bash
-   wget https://raw.githubusercontent.com/grafana/loki/main/examples/getting-started/loki-config.yaml -O loki-config.yaml
-   wget https://raw.githubusercontent.com/grafana/loki/main/examples/getting-started/alloy-local-config.yaml -O alloy-local-config.yaml
-   wget https://raw.githubusercontent.com/grafana/loki/main/examples/getting-started/docker-compose.yaml -O docker-compose.yaml
-   ```
-   {{< /docs/ignore >}}
-
-3. Deploy the sample Docker image.
-
-   With `evaluate-loki` as the current working directory, start the demo environment using `docker compose`:
-
-   ```bash
-   docker compose up -d
-   ```
-
-   At the end of the command, you should see something similar to the following:
-
-   ```console
-   ✔ Network evaluate-loki_loki          Created      0.1s
-   ✔ Container evaluate-loki-minio-1     Started      0.6s
-   ✔ Container evaluate-loki-flog-1      Started      0.6s
-   ✔ Container evaluate-loki-backend-1   Started      0.8s
-   ✔ Container evaluate-loki-write-1     Started      0.8s
-   ✔ Container evaluate-loki-read-1      Started      0.8s
-   ✔ Container evaluate-loki-gateway-1   Started      1.1s
-   ✔ Container evaluate-loki-grafana-1   Started      1.4s
-   ✔ Container evaluate-loki-alloy-1     Started      1.4s
-   ```
-
-
-4. (Optional) Verify that the Loki cluster is up and running.
-
-   - The read component returns `ready` when you browse to [http://localhost:3101/ready](http://localhost:3101/ready).
-     The message `Query Frontend not ready: not ready: number of schedulers this worker is connected to is 0` shows until the read component is ready.
-   - The write component returns `ready` when you browse to [http://localhost:3102/ready](http://localhost:3102/ready).
-     The message `Ingester not ready: waiting for 15s after being ready` shows until the write component is ready.
-
-5. (Optional) Verify that Grafana Alloy is running.
-   - You can access the Grafana Alloy UI at [http://localhost:12345](http://localhost:12345).
-
-6. (Optional) You can check all the containers are running by running the following command:
-
-   ```bash
-   docker ps -a
-   ```
+**To deploy the Loki Stack locally, follow these steps:**
 
+1. Clone the Loki fundamentals repository and checkout the getting-started branch:
+
+     ```bash
+     git clone https://github.com/grafana/loki-fundamentals.git -b getting-started
+     ```
+
+1. Change to the `loki-fundamentals` directory:
+
+     ```bash
+     cd loki-fundamentals
+     ```
+
+1. With `loki-fundamentals` as the current working directory deploy Loki, Alloy, and Grafana using Docker Compose:
+
+     ```bash
+     docker compose up -d
+     ```
+      After running the command, you should see a similar output:
+
+      ```console
+       ✔ Container loki-fundamentals-grafana-1  Started  0.3s 
```

Comment:
I'm not seeing the Drilldown menu item in Killercoda. Drilldown is still under Explore > Logs.  And the direct link at line 103 gives a 404.

And when I run it on my local, I'm getting the same result.  We've somehow lost the Drilldown menu that we had the last time I ran through this.